integrationTests/src/test/resources/multi-module-with-submodules/settings.gradle.kts

```diff
@@ -0,0 +1,20 @@
+pluginManagement {
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id == "org.sonarqube") {
+                useModule("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:${requested.version}")
+            }
+        }
+    }
+    repositories {
+        mavenLocal()
+        maven(url="maven-repo")
+        gradlePluginPortal()
+        ivy("ivy-repo")
+    }
+}
+
+include("module")
+include("skippedModule")
+include("module:submodule")
+include("skippedModule:skippedSubmodule")
```

Comment:
So we have confirmation that we need to add submodule to the top level project?

Reply:
Good question! You can include the modules in any `settings.gradle`, but you need to be careful because a module will apply the settings only from its closest ancestor!
In this case is fine to have a single settings file at the root and thus include all the modules there